baselines/bigquery-storage/test/gapic-big_query_storage-v1beta1.ts.baseline

```diff
@@ -19,6 +19,7 @@
 import * as protosTypes from '../protos/protos';
 import * as assert from 'assert';
 import { describe, it } from 'mocha';
+/* eslint-disable @typescript-eslint/no-var-requires */
 const bigquerystorageModule = require('../src');
```

Comment:
This is a good time to ask ...  why is this a `require` and not an `import`?

Reply:
import module will bring the types in :( 
we have many mock methods like `client._innerApiCalls.echo` but in EchoClient `_innerApiCalls` is private, which will throw type checking error.

Reply:
@alexander-fenster also brought this up in this pr: https://github.com/googleapis/gapic-generator-typescript/pull/354

Reply:
spent some time to convert `require` to `import`, it needs more changes than just making `_innerApiCalls ` public. LROperation, gax.CallOptions ... are incompatible with the real types of method calls. Some of them can be forced compatible but lots of `as unknown as someType` will be used.  How about we leave it as it is for now, and as @alexander-fenster suggested, move unit test to `sinon` for removing custom mock codes later as a separate PR.